homeworks/oleksandr.khodakivskyi_Moo0oony/4-basics/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Basic math operations</title>
+        <link rel="stylesheet" href="./style.css">
+    </head>
+    <body>
+        <main>
+            <h1>Basic math operations</h1>
+            <p>Open console to see the results</p>
+            <article id="operations"></article>
```

Comment:
Every `article` should have a header h1-h6 - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article#usage_notes
So, probably this is a `section`